pinot-query-runtime/src/main/java/org/apache/pinot/query/mailbox/GrpcSendingMailbox.java

```diff
@@ -74,16 +78,17 @@ public void complete() {
   }
 
   @Override
-  public void cancel(Throwable t) {
+  public void cancel(@Nullable Throwable t) {
```

Comment:
There were some arguments and parameters that could be null but were not marked as such. I tried to mark the ones I discovered.